tests/compile-test.rs

```diff
@@ -296,7 +300,7 @@ fn run_ui_cargo() {
             config.out_dir = PathBuf::from("target/ui_test_cargo/").join(path.parent().unwrap());
             Some(config)
         },
-        (status_emitter::Text, status_emitter::Gha::<true> { name }),
+        status_emitter::Text,
```

Comment:
@oli-obk so should we apply the same diff in Miri to reduce the confusing output there?

Reply:
We can also just fix it ^^

Reply:
Once it is Just Fixed :tm: , we'll also revert this and re-enable it in Clippy :)